src/overlays/actors/ovl_En_Kusa/z_en_kusa.c

```diff
@@ -270,7 +267,6 @@ void EnKusa_UpdateVelY(EnKusa* this) {
 
 /**
  * @brief Scales a vector down by provided scale factor
- *
  * @param vec   vector to be scaled
  * @param scaleFactor   scale factor to be applied to vector
```

Comment:
I meant between the argument name and the description, it's fine to have a blank line before arguments.